index.html

```diff
@@ -1891,7 +1893,180 @@ <h4>Create Workflow</h4>
+      "verifiablePresentationRequest": {
+        "query": {
+          "type": "DIDAuthentication",
+          "acceptedMethods": [
+            {
+              "method": "key"
+            },
+            {
+              "method": "web"
+            }
+          ],
+          "acceptedCryptosuites": [
+            {
+              "cryptosuite": "eddsa-rdfc-2022"
+            },
+            {
+              "cryptosuite": "ed25519-2020"
+            }
+          ]
+        },
+        "domain": "https://issuer.example.com"
+      },
+      "nextStep": "credentialDelivery"
+    },
+    "credentialDelivery": {
+      "issueRequests": [
+        {
+          "credentialTemplateId": "caffb919-053a-4bfa-beba-80567904f842",
+          "variables": "sampleAchievementCredential"
+        }
+      ]
+    }
+  },
+  "credentialTemplates": [
+    {
+      "id": "caffb919-053a-4bfa-beba-80567904f842",
+      "type": "jsonata",
+      "template": "{
+        "@context": [
+          "https://www.w3.org/ns/credentials/v2",
+          "https://purl.imsglobal.org/spec/ob/v3p0/context-3.0.3.json",
+          "https://purl.imsglobal.org/spec/ob/v3p0/extensions.json"
+        ],
+        "id": sampleAchievementCredential.id,
+        "type": ["VerifiableCredential", "AchievementCredential"],
+        "issuer": {
+          "type": ["Profile"],
+          "name": "Example Issuer",
+          "image": {
+            "type": "Image",
+            "id": "https://issuer.example.com/logo.png"
+          }
+        },
+        "awardedDate": sampleAchievementCredential.awardedDate,
+        "validFrom": sampleAchievementCredential.validFrom,
+        "name": "Sample Achievement",
+        "credentialSubject": {
+          "id": results.didAuthRequest.verifiablePresentation.holder,
+          "type": "AchievementSubject",
+          "name": sampleAchievementCredential.credentialSubject.name,
+          "achievement": {
+            "id": sampleAchievementCredential.credentialSubject.achievement.id,
+            "type": ["Achievement"],
+            "achievementType": sampleAchievementCredential.credentialSubject.achievement.achievementType,
+            "name": sampleAchievementCredential.credentialSubject.achievement.name,
+            "description": sampleAchievementCredential.credentialSubject.achievement.description,
+            "alignment": sampleAchievementCredential.credentialSubject.achievement.alignment
+          }
+        }
+      }"
+    }
+  ],
+  "controller": "did:web:issuer.example.com"
+}
+        </pre>
+
+        <p>
+Some workflows will need to reference custom variables in one or more
+of their steps. In this case, the coordinator will need
+to provide the appropriate value(s) for the variable(s) when it
+creates an exchange. Here is an example of a workflow configuration
+with a templated DID Authentication step
+(Notice the `verifiablePresentationRequest` and `callbackUrl` variables
+referenced in the `didAuthRequest` step):
```

Comment:
@dlongley Do you think this is a decent enough explanation/motivation for the custom variables given our discussions in this PR and elsewhere?